pkg/common/constants.go

```diff
@@ -11,6 +11,9 @@ const (
 	// PVNameKey contains name of the final PV that will be used for the dynamically
 	// provisioned volume
 	PVNameKey = "csi.storage.k8s.io/pv/name"
+
+	// ClusterIDKey contains the cluster id key of the PV cities in
+	ClusterIDKey = "csi.storage.k8s.io/clusterid"
```

Comment:
This should be added after line 46.
Maybe naming it `csi.alibabacloud.com/cluster-id` ? I think we should not use k8s.io without the consensus with the community.